tests/main/views/test_create_user.py

```diff
@@ -121,26 +121,74 @@ def test_should_render_correct_error_page_if_token_expired(
                 t="send a new one",
             )
 
-    def test_should_render_create_user_page_if_user_does_not_exist(self):
+    def test_should_render_create_user_page_if_supplier_does_not_exist(self):
```

Comment:
Do we need to test buyers and suppliers separately? Can we parametrize instead?

Reply:
I wanted to minimise the logic in the test, if you think it would be easier to read though I will change it

Reply:
Is it just the phone number that's different? I think that's fine to have in one test

Reply:
Done